qiskit_experiments/library/stark/ramsey_xy.py

```diff
@@ -0,0 +1,319 @@
+        Ramsey sequence against the shifted frequency.
+        This frequency shift is a function of the offset of the Stark tone from the
+        qubit frequency, and magnitude of the tone.
+
+        Note that the Stark tone takes the form of the flat-topped Gaussian envelope,
+        in which the magnitude of pulse varies in time in rising and falling edges,
+        resulting in the time-dependent Stark shift.
+        This is hardly characterized by the Ramsey experiment, and thus an additional pulse
+        involving only edges (StarkV) is added in front of the primary Stark drive (StarkU).
+        The sign of phase accumulation is inverted with the X gate in the middle to
+        cancel out the unwanted phase accumulation in the pulse edges.
+
+        This technique allows the experiment to only accumulate the net phase
+        during the flat-top part of the StarkU pulse with the constant magnitude.
+
+    # section: note
+
+        In the lower power limit, the Stark shift has a quadratic dependency on the
+        pulse amplitude. This implies the resulting Stark shift is not sensitive to
+        the sign of the amplitude of StarkV and StarkU pulses on one hand.
+        On the other hand, the sign of the shift depends on the sign of frequency offset.
+        In other words, positive (negative) Stark shift occurs when the Stark tone frequency
+        is lower (higher) than the qubit frequency.
+
+        In this experiment, the amplitude of Stark tone, i.e. `stark_amp` parameter, is
+        abstracted to give positive (positive) Stark shift with positive (negative) stark_amp
+        for better handling of the experiment parameters.
+        The StarkV and StarkU pulses are always positive regardless of the sign of stark_amp,
+        while the sign of frequency offset is implicitly determined by its sign.
+
+    # section: analysis_ref
+        :py:class:`RamseyXYAnalysis`
+
+    # section: see_also
+        qiskit_experiments.library.characterization.ramsey_xy.RamseyXY
+
+    """
+
+    def __init__(
+        self,
+        qubit: int,
+        stark_amp: float,
+        backend: Optional[Backend] = None,
+        **experiment_options,
+    ):
+        """Create new experiment.
+
+        .. note::
+
+            .. _stark_amplitude:
+
+            The `stark_channel` in the experiment options may be necessary supplied.
+            By default, the Stark tone is applied to the same channel with the qubit drive
+            with frequency modulation. This frequency modulation might update the
+            channel frame, which accumulates unwanted phase against the frequency difference
+            between the qubit drive and Stark tone frequency in addition to
+            the qubit Stark shift. You can use a dedicated Stark drive channel if available,
+            otherwise you may want to reuse a control channel associated with the qubit.
+            This depends on the architecture of your quantum device.
```

Comment:
```suggestion
            It may be necessary to supply the ``stark_channel`` experiment option.
            By default, the Stark tone is applied to the same channel as the qubit drive
            with a frequency shift. This frequency shift might update the
            channel frame, which accumulates unwanted phase against the frequency difference
            between the qubit drive and Stark tone frequencies in addition to
            the qubit Stark shift. You can use a dedicated Stark drive channel if available.
            Otherwise you may want to use a control channel associated with the physical
            drive port of the qubit (e.g. in a cross-resonance system, a control channel for
            which the qubit is the control qubit).
            This depends on the architecture of your quantum device.
```